guide/geometries.md

```diff
@@ -5,14 +5,97 @@ The world files for these examples are located in the "sample/geometries/worlds"
 
 In this directory, you will find the following world files :
 
-- box.wbt
-- cone.wbt
-- convex\_polygon.wbt
-- cylinder.wbt
-- high\_resolution\_indexedfaceset.wbt
-- non\_convex\_polygon.wbt
-- physics\_primitives.wbt
-- polyhedra.wbt
-- sphere.wbt
-- textured\_shapes.wbt
-- webots\_box.wbt
+### box.wbt, capsule.wbt, cone.wbt, convex\_polygon.wbt, cylinder.wbt, non\_convex\_polygon.wbt, polyhedra.wbt, sphere.wbt
+
+**Keywords**: Box, Capsule, Cone, Cylinder, Sphere, IndexedFaceSet, basic 3D primitives
+
+These examples show how to use the basic geometric primitives of Webots in order to display untextured and colored 3D shapes in the 3D window.
+For the most part, primitives are static in these examples, i.e. physics is not applied on them.
+Basically :
+
+- `box.wbt` demonstrates how to use the `Box` node to display a red box.
+- `capsule.wbt`: demonstrates how to use the `Capsule` node to display a red capsule.
+- `cone.wbt`: demonstrates how to use the `Cone` node to display a red cone.
+- `convex_polygon.wbt`: demonstrates how to use the `IndexedFaceSet` node to display a convex polygon.
+- `cylinder.wbt`: demonstrates how to use the `Cylinder` node to display a red cylinder.
+- `polyhedra.wbt`: demonstrates how to use the `IndexedFaceSet` node to display a red polyhedra.
+- `sphere.wbt`: demonstrates how to use the `Sphere` node to display a red sphere.
+
+### extended\_solids.wbt
+
+**Keywords**: Extended solids, torus, rounded box, pipe
+
+![extended_solids.png](images/extended_solids.png) This example demonstrates the use of several extended primitives.
+These compound primitives are created using procedural PROTOs that generate a set of basic Geometry nodes.
```

Comment:
We do not use PROTOs in the documentation. Instead we write PROTO nodes, PROTO files, etc.